src/components/data_submitter/DataAccessGovernance.js

```diff
@@ -0,0 +1,168 @@
+  fontFamily: 'Montserrat',
+  fontSize: '14px',
+};
+const radioContainer = {
+  marginTop: '.5rem',
+};
+
+export const DataAccessGovernance = (props) => {
+  const {
+    formData, setFormData
+  } = props;
+
+  const [consentGroups, setConsentGroups] = useState([]);
+  const [dacs, setDacs] = useState([]);
+
+
+  const updateFormData = (updatedFields) => {
+    setFormData({
+      ...formData,
+      ...updatedFields,
+    });
+  };
+
+  useEffect(() => {
+    updateFormData({
+      consentGroups: consentGroups.filter((val) => !isNil(val)),
+    })
+  }, [consentGroups]);
+
+  useEffect(() => {
+    DAC.list(false).then((dacList) => {
+      setDacs(dacList);
+    });
+  }, [])
+
+  const addNewConsentGroup = () => {
+    const newConsentGroups = cloneDeep(consentGroups);
+    newConsentGroups.push({});
+    setConsentGroups(newConsentGroups)
+  };
+
+  const deleteConsentGroup = (idx) => {
+    const newConsentGroups = cloneDeep(consentGroups);
+    newConsentGroups[idx] = undefined;
+    setConsentGroups(newConsentGroups);
+  };
+
+  const updateConsentGroup = (idx, update) => {
+    const newConsentGroups = cloneDeep(consentGroups);
+    newConsentGroups[idx] = {
+      ...consentGroups[idx],
+      ...update,
+    };
+    updateFormData({
+      consentGroups: newConsentGroups,
+    });
+  };
+
+
+
+  return div({}, [
+    div({}, [
+
+      // open or closed access
+      div({
+        style: radioContainer,
+      }, [
+        RadioButton({
+          style: radioButtonStyle,
+          id: 'alternativeDataSharingPlanControlledOpenAccess',
+          name: 'alternativeDataSharingPlanControlledOpenAccess',
+          value: 'alternativeDataSharingPlanControlledOpenAccess',
+          defaultChecked: formData.alternativeDataSharingPlanControlledOpenAccess === OPEN_ACCESS,
+          onClick: () => updateFormData({
+            alternativeDataSharingPlanControlledOpenAccess: OPEN_ACCESS,
+          }),
+          description: 'Open Access',
+        }),
+      ]),
+
+      div({
+        style: radioContainer,
+      }, [
+        RadioButton({
+          style: radioButtonStyle,
+          id: 'alternativeDataSharingPlanControlledOpenAccess',
+          name: 'alternativeDataSharingPlanControlledOpenAccess',
+          value: 'alternativeDataSharingPlanControlledOpenAccess',
+          defaultChecked: formData.alternativeDataSharingPlanControlledOpenAccess === CLOSED_ACCESS,
+          onClick: () => updateFormData({
+            alternativeDataSharingPlanControlledOpenAccess: CLOSED_ACCESS,
+          }),
+          description: 'Closed Access',
+        }),
+      ]),
+
+      
+      div({
+        isRendered: formData.alternativeDataSharingPlanControlledOpenAccess === CLOSED_ACCESS,
+      },
+        [
+          // select dac
+
+          h(SearchSelect, {
+            id: 'data_submission_select_dac',
+            name: 'data_submission_select_dac',
+            onSelection: (selection) => updateFormData({
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 10 spaces but found 12.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9969451)
```suggestion
          onSelection: (selection) => updateFormData({
```